pywb/warcserver/inputrequest.py

```diff
@@ -228,9 +228,18 @@ def __init__(self, method, mime, length, stream,
         if not mime:
             mime = ''
 
-        if mime.startswith('application/x-www-form-urlencoded'):
+        def handle_binary(query):
+            query = base64.b64encode(query)
             query = to_native_str(query)
-            query = unquote_plus(query)
+            query = '__wb_post_data=' + query
+            return query
+
+        if mime.startswith('application/x-www-form-urlencoded'):
+            try:
+                query = to_native_str(query.decode())
```

Comment:
```suggestion
                query = to_native_str(query.decode('utf-8'))
```
Yep this makes sense! Want to ensure checking for utf-8, so adding it explicitly

Reply:
Righto.